wgpu-types/src/limits.rs

```diff
@@ -573,7 +574,7 @@ impl Limits {
             max_compute_workgroups_per_dimension: 0,
 
             // Value supported by Intel Celeron B830 on Windows (OpenGL 3.1)
```

Comment:
```suggestion
            // Intel Celeron B830 on Windows (OpenGL 3.1) supports up to 31 bytes (~8 variables)
```

(Or delete it entirely. It feels like a shame to lose the research but it may not be worth maintaining.)

Reply:
I don't know if this is bytes and words, or words and vec4s.

Reply:
@kpreid is the author of this code originally, so let's ping him and ask! Kevin, Do you recall what the actual capabilities are here? If we need to, then this is an interesting conflict with the spec., in which even compatibility mode only lowers the minimum support required to 15. I'm guessing that, as @andyleiserson suspects, we should only allow 8 variables here?

I'll file an issue and follow-up if there are changes we need to make here before releasing. I'm still gonna merge, tho. :shipit: